npm-packages/eslint-plugin-meteor/scripts/dev-bundle-tool-package.js

```diff
@@ -14,8 +14,8 @@ var packageJson = {
     pacote: "https://github.com/meteor/pacote/tarball/a81b0324686e85d22c7688c47629d4009000e8b8",
     "node-gyp": "8.0.0",
     "node-pre-gyp": "0.15.0",
-    typescript: "4.5.4",
-    "@meteorjs/babel": "7.16.0-beta.1",
+    typescript: "4.6.4",
```

Comment:
As I mentioned in https://github.com/meteor/meteor/pull/12313 we don't need this scripts folder and we need to update `scripts/dev-bundle-tool-package.js` instead of `npm-packages/eslint-plugin-meteor/scripts/dev-bundle-tool-package.js`

Reply:
@Grubba27 Meteor itself doesn't use new TypeScript because of the issue I mentioned above.

Reply:
Did not realize that. Thanks again for pinging me on this.
Do you want to make a pr for this? I will add it to the 2.9.1 patch release. If not, I will do it